packages/patternfly-3/patternfly-react-extensions/less/diffview.less

```diff
@@ -0,0 +1,41 @@
+@import '~react-diff-view/index.css';
+
+.diff {
+  border: 1px solid #d1d1d1;
```

Comment:
Colors should use `color-pf-...` variables

Reply:
see: https://www.patternfly.org/styles/color-palette/
and use it as `$color-pf-black-200` for example